app/models/system_console.rb

```diff
@@ -53,10 +53,10 @@ def self.launch_proxy(remote_address, remote_port)
       return nil
     end
 
-    command = '/usr/bin/socat', "TCP-LISTEN:#{local_port},fork", "TCP:#{remote_address}:#{remote_port}"
+    command = ["/usr/bin/socat", "TCP-LISTEN:#{local_port},fork", "TCP:#{remote_address}:#{remote_port}"]
```

Comment:
instead of creating 1 array, and then splitting out parts, it may be easier to just create an `args` variable

```suggestion
    args = ["TCP-LISTEN:#{local_port},fork", "TCP:#{remote_address}:#{remote_port}"]
```

not sure if you create `command = "/usr/bin/socat"` or if you just pass directly into spawn